src/local_sensitivity/backsolve_adjoint.jl

```diff
@@ -81,6 +82,76 @@ end
   return ODEProblem(odefun,z0,tspan,p,callback=cb)
 end
 
+
+
+@noinline function SDEAdjointProblem(sol,sensealg::BacksolveAdjoint,
+                                     g,t=nothing,dg=nothing;
+                                     checkpoints=sol.t,
+                                     callback=CallbackSet(),kwargs...)
+  @unpack f, p, u0, tspan = sol.prob
+  tspan = reverse(tspan)
+  discrete = t != nothing
+
+  p === DiffEqBase.NullParameters() && error("Your model does not have parameters, and thus it is impossible to calculate the derivative of the solution with respect to the parameters. Your model must have parameters to use parameter sensitivity calculations!")
+  numstates = length(u0)
+  numparams = length(p)
+
+  len = length(u0)+numparams
+  λ = similar(u0, len)
+
+  drift_function = ODEFunction(sol.prob.f)
```

Comment:
that's a little awkward. How come this is required?

Reply:
I needed it for the diffusion.. to get it through the Jacobian checks and thought it might be better to have it symmetric with respect to the drift  (though there it is indeed not required)

Reply:
`sol.prob.f` is already an ODEFunction though?